packages/odyssey-storybook/src/components/Field/Field.mdx

```diff
@@ -1,43 +1,150 @@
+import { Canvas, Story, ArgsTable } from "@storybook/addon-docs";
+
 # Field
 
-These captions help make forms more accessible by providing context to the user. They can be used with all Odyssey inputs.
+Field provides all of the supporting context required by inputs, including labels, hints, and errors.
+
+This component is also responsible for managing the appropriate application of `aria-` attributes and `id` associations between Inputs and their associated elements.
+
+[View source on GitHub](https://github.com/okta/odyssey/tree/master/packages/odyssey-react/src/components/Field)
 
 ## Anatomy
 
-**STORY MISSING**
+<Canvas>
+  <Story id="components-textinput--kitchen-sink" />
+</Canvas>
+
+### Label
+
+A text label for the Field. The associated element may be `<label>` or `<legend>` depending on Field's Input type.
+
+<Canvas>
+  <Story id="components-field--field-label" />
+</Canvas>
+
+The label also indicates the Field's optional/required status to ensure screen readers properly announce the state.
+
+<Canvas>
+  <Story id="components-field--field-optional" />
+</Canvas>
+
+### Hint
+
+A hint is helper text for the Field. The field hint is similar to the disallowed `placeholder` attribute. It should provide any necessary help text for the Field but is optional.
+
+Appropriate examples include expected formatting or value types.
+
+<Canvas>
+  <Story id="components-field--field-hint" />
+</Canvas>
+
+### Error
+
+Inline error messaging for the Field.
+
+All errors require a visible or SR-only "Error:" label. This labeling is not handled automatically and must be included in the error message.
+
+<Canvas>
+  <Story id="components-field--field-error" />
+</Canvas>
+
+## Props
+
+<ArgsTable />
+
+## Variants
+
+While their appearance is identical, Field supports single and grouped inputs by offering an `as=" foo"` polymorphic API.
+
+### Single Input
+
+Fields for lone inputs are rendered as `div's and assign their `label`/`input` relationship using an ID.
+
+<Canvas>
+  <Story id="components-textarea--default" />
+</Canvas>
+
+### Grouped Inputs
+
+Fields for grouped inputs are rendered as `fieldset`s, using a `legend` to label all of the associated `input`s accessibly. This variant also spreads hint and error associations across affected controls.
+
+<Canvas>
+  <Story id="components-radio--default" />
+</Canvas>
+
+## States
+
+Fields may be either required or optional, depending on what's called for by their input.
+
+### Required
+
+By default, inputs and Fields are set to `required`. As the default, this state has no unique affordance.
+
+<Canvas>
+  <Story id="components-textinput--text" />
+</Canvas>
+
+### Optional
+
+Fields with `required` set to `false` will be marked as optional:
```

Comment:
consider rewording to: explicitly set to false. It is the only way to mark a field optional